Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -140,6 +140,11 @@
 from Configuration.Eras.Modifier_phase2_timing_layer_bar_cff import phase2_timing_layer_bar
 (phase2_timing_layer_tile | phase2_timing_layer_bar).toReplaceWith(globalreco,_phase2_timing_layer_globalreco)
 
+_phase2_timing_layer_globalreco_tracking = globalreco_tracking.copy()
+_phase2_timing_layer_globalreco_tracking += fastTimingGlobalReco
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toReplaceWith(globalreco_tracking,_phase2_timing_layer_globalreco_tracking)
```

Comment:
shouldn't the piece above with ```.toReplaceWith(globalreco,_phase2_timing_layer_globalreco)``` be removed?

Reply:
also, it would be better to modify ```globalreco_tracking``` right after its definition instead of after some other sequences

Reply:
@slava77 I see your point, as it is now ```fastTimingGlobalReco``` appears twice inside teh globalreco sequence. As far as I can see this module should be independent on the other parts of globalreco, so it should work

Reply:
@slava77 I agree with both of your comments

Reply:
from https://github.com/cms-sw/cmssw/pull/27529/files#r303926710


> also, it would be better to modify `globalreco_tracking` right after its definition instead of after some other sequences

Reply:
to be more clear, I'm suggesting to 
- move the currently proposed L138,139,140-143 to L122
- remove everything else from L137 to L144

Reply:
I think you meant 141 - 143, 140 at this point should just go together with 137